src/components/LinkCard/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Flex, View, Link } from '@aws-amplify/ui-react';
+import { ExternalLinkIcon } from '../Icons';
+import styles from './link-card.module.css';
+
+export interface CardData {
```

Comment:
Looks like this export isn't needed/used in this file and can be removed?

Reply:
I have added this for testing purpose, I will remove this, thanks